docs/rst/tutorials/tools/monitor_with_backend/monitor_with_backend.rst

```diff
@@ -0,0 +1,203 @@
+    :backlinks: none
+
+Background
+----------
+
+Vulcanexus integrates :ref:`vulcanexus_monitor`, which is a useful tool for monitoring and studying a ROS 2 network as ROS 2 relies on the `DDS specification <https://www.omg.org/spec/DDS/1.4/About-DDS/>`_ to communicate the different nodes.
+
+This tutorial provides an explanation of the ROS 2 Monitor code.
+
+Prerequisites
+-------------
+
+It is required to have previously installed Vulcanexus using one of the following installation methods:
+
+* :ref:`linux_binary_installation`
+* :ref:`linux_source_installation`
+* :ref:`docker_installation`
+
+Ensure that the Vulcanexus installation includes Vulcanexus Tools (either ``vulcanexus-humble-desktop``, ``vulcanexus-humble-tools``, or ``vulcanexus-humble-base``).
+
+Create the application workspace
+--------------------------------
+
+The application workspace will have the following structure at the end of the project.
+
+.. code-block:: shell-session
+
+    .
+    └── monitor
+        ├── include
+        │   ├── monitor
+        └── src
+            └── monitor.cpp
+        ├── CMakeLists.txt
+        └── package.xml
+
+Let's create the ROS 2 package by running the command:
+
+.. code-block:: bash
+
+    ros2 pkg create --build-type ament_cmake monitor --dependencies fastcdr fastrtps fastdds_statistics_backend
+
+You will now have a new folder within your workspace ``src`` directory called ``monitor``.
+
+Import linked libraries and its dependencies
+--------------------------------------------
+
+The DDS application requires the Fast DDS and Fast CDR libraries.
+Depending on the installation procedure followed the process of making these libraries available for our DDS application
+will be slightly different.
+
+Installation from binaries and manual installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If we have followed the installation from binaries or the manual installation, these libraries are already accessible
+from the workspace.
+On Linux, the header files can be found in directories `/usr/include/fastrtps/` and
+`/usr/include/fastcdr/` for Fast DDS and Fast CDR respectively. The compiled libraries of both can be found in
+the directory `/usr/lib/`.
+
+Colcon installation
+^^^^^^^^^^^^^^^^^^^
+
+From a Colcon installation there are several ways to import the libraries.
+If the libraries need to be available just for the current session, run the following command.
+
+.. code-block:: bash
+
+    source <path/to/Fast-DDS/workspace>/install/setup.bash
+
+They can be made accessible from any session by adding the Fast DDS installation directory to your ``$PATH``
+variable in the shell configuration files for the current user running the following command.
+
+.. code-block:: bash
+
+    echo 'source <path/to/Fast-DDS/workspace>/install/setup.bash' >> ~/.bashrc
+
+This will set up the environment after each of this user's logins.
+
+Write the Monitor
+-----------------
+
+This is the C++ source code for the application.
```

Comment:
Explain here that the user should download this file from somewhere. As long as you will not have the link until this PR is merged, leave the sentence prepare to insert the actual link in another PR.